database/migrations/2021_12_04_181200_refactor_models.php

```diff
@@ -280,7 +280,7 @@ public function up()
 		$this->createSizeVariantTableUp();
 		$this->createSymLinkTableUp();
 		$this->createRemainingForeignConstraints();
-		$this->createWebAuthnTableUp();
+		// $this->createWebAuthnTableUp();
```

Comment:
If code isn't needed anymore or became even wrong/harmful/whatever, delete it properly. Or if there is a chance that his code might be needed again in the future, add a comment. But otherwise everyone will wonder in the future why this code has been commented out.

Reply:
fixed [faf44a2](https://github.com/LycheeOrg/Lychee/pull/1469/commits/faf44a2f63086be3902560abe44d2c11a8daf791)

Reply:
I will keep this open for the moment until @kamil4 and @d7415 have given their thoughts on https://github.com/LycheeOrg/Lychee/pull/1469#discussion_r985245067.

I am not in favour of fiddling with old migrations for a little efficiency gain as I deem the risk of introducing new lurking bugs as too high. For fresh installations all the tables are empty anyway so it is not much what we gain here.

Reply:
fixed [3173156](https://github.com/LycheeOrg/Lychee/pull/1469/commits/317315682bd1bc5e5c95bc2f959adde8cff2d402)